components/widgets/resources/src/main/resources/resources/uicomponents/PhenotypeSelectionUtils.js

```diff
@@ -41,7 +41,7 @@ define("PhenotypeSelectionUtils", [], function() {
            return null;
         }
         var id = phenotype.id;
-        var searchUrl = new XWiki.Document('SolrService', 'PhenoTips').getURL("get", "q=" + id);
+        var searchUrl = XWiki.contextPath + "/rest/vocabularies/hpo/" + id;
```

Comment:
scratch that, it is `hpo/suggest?input=` and `hpo/id`, that makes sense. Just the old code seems to used the same URL schema for both searches and queries of a single term by ID